tools/cldr-code/src/main/java/org/unicode/cldr/util/GrammarInfo.java

```diff
@@ -755,7 +755,11 @@ public static Set<String> getGrammarLocales() {
                     "knot",
                     "dalton",
                     "kilocalorie",
-                    "electronvolt");
+                    "electronvolt",
+                    // The following may be reinstated after 45.
```

Comment:
need a TODO ticket link for this?

Reply:
Filed yesterday: https://unicode-org.atlassian.net/browse/CLDR-17459

Reply:
good, should ideally be in code